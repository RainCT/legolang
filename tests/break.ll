for i in 1 .. 15:
    print i
    if i % 10 == 6:
        break

print ""

string[] xs
xs[] = "aa"
xs[] = "bb"
xs[] = "cc"

foreach x in xs:
    print x
    foreach y in xs:
        for i in 3 .. 30:
            print y
            break
        break

print ""
print "out"
