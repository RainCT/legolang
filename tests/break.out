1
2
3
4
5
6

aa
aa
bb
aa
cc
aa

out
